### DISCUSSION TITLE
Erro ao validar Token

### DISCUSSION THREAD

**Discussion by MARQUESEFERREIRA** [HashLoad/horse#301] [Q&A] [answered]
Além de não conseguir puxar o Acces e Refresh está dando erro nestas linhas abaixo que não estou conseguindo avançar na aula...                   [dcc32 Error] Controllers.Auth.pas(67): E2034 Too many actual parameters
[dcc32 Error] Controllers.Auth.pas(67): E2035 Not enough actual parameters
[dcc32 Fatal Error] auth.dpr(13): F2063 Could not compile used unit 'Controllers.Auth.pas'

**Comment by viniciussanchez** [comment, marked as answer]
@MARQUESEFERREIRA primeiramente, você precisa ser mais específico. Aqui não é o lugar certo para tirar dúvidas de curso. Até porque existem vários cursos de Horse, de várias pessoas, e da forma que colocou, seria impossível saber. Segundo, a mensagem fala que você está passando uma quantidade de parâmetros para uma função, no qual não deveria. Provavelmente você deve estar fazendo coisa errada ai. O processo em si funciona perfeitamente.
